docs/pages/development/raster_v1.rst

```diff
@@ -0,0 +1,253 @@
+   aerials/north-head/.raster-dataset.v1/meta/format.json        # Format of the dataset
+   aerials/north-head/.raster-dataset.v1/meta/schema.json        # Schema of the dataset
+   aerials/north-head/.raster-dataset.v1/meta/crs.wkt            # CRS of the dataset
+   aerials/north-head/.raster-dataset.v1/meta/band-1/...         # Extra metadata specific to the first band...
+   aerials/north-head/.raster-dataset.v1/meta/band-2/...         # Extra metadata specific to the second band...
+   aerials/north-head/.raster-dataset.v1/meta/...                # Other dataset metadata
+
+   aerials/north-head/.raster-dataset.v1/tile/...                # All of the tiles in the dataset - all GeoTIFFs
+
+Meta items
+~~~~~~~~~~
+
+The following items are stored in the meta part of the dataset, and have the following structure.
+
+``meta/title``
+^^^^^^^^^^^^^^
+
+Contains the title of the dataset, encoded using UTF-8. The title is freeform text, clients could if they desired include HTML tags or markdown in the title.
+
+``meta/description``
+^^^^^^^^^^^^^^^^^^^^
+
+Contains the title of the dataset, encoded using UTF-8. The description is freeform text, clients could if they desired include HTML tags or markdown in the title.
+
+``meta/format.json``
+^^^^^^^^^^^^^^^^^^^^
+
+Contains information about constraints related to file format that the GeoTIFF tiles must follow. A single dataset cannot contain a mixture of all kinds of GeoTIFF files - there must be some commonality between them. For example, every tile in a dataset must have the same CRS.
+
+In terms of format, currently there are only one or two constraints stored in the format.json that every tile in the dataset must follow:
+1. All tiles must be GeoTIFF (this constraint is always present for Raster Datasets V1 - no other file types are supported).
+2. All tiles must be `Cloud-Optimized GeoTIFFs <cog_>`_. This constraint is only set for certain datasets where the user has requested it.
+
+Constraining the tiles to be Cloud Optimized GeoTIFFs (COGs) means they can be viewed by webviewers - the COG format is a subtype of GeoTIFF file which allows for fasting seeking and rendering of whichever part of the raster is currently within the user's viewport.
+
+For example, here is the format of a dataset that has both constraints:
+
+.. code:: json
+
+    {
+      "fileType": "geotiff",
+      "profile": "cloud-optimized"
+    }
+
+If the cloud-optimized constraint is relaxed, such that both COGs and non-COGs are allowed, then the relevant field is ommitted.
+
+.. _raster-meta-schema-json:
+
+``meta/schema.json``
+^^^^^^^^^^^^^^^^^^^^
+
+Contains information about the "bands" of information, that is, each unit of data that is stored per-pixel. This item has a similar format as in other types of datasets, but one important difference is that the each band in a raster dataset has no identifying ID or name - the properties that identify a raster band are the following:
+
+1. Its position in the list.
```

Comment:
```suggestion
1. It's position in the list.
```

Reply:
Not done - "its" is actually correct.
https://www.merriam-webster.com/words-at-play/when-to-use-its-vs-its

Reply:
ok - this one's pretty confusing though hey
maybe fix line 74 then?

Reply:
Good point. I can only see snippets of what I wrote in this review-conversation so can't see the line right next to it.

Reply:
Done